src/experimental/create.tsx

```diff
@@ -35,5 +40,12 @@ export const create = <PeerMetadata, TrackMetadata>(): CreateNoContextJellyfishC
     useSelector: <Result,>(selector: Selector<PeerMetadata, TrackMetadata, Result>): Result => {
       return useSelector(store, selector);
     },
+    /* eslint-disable react-hooks/rules-of-hooks */
+    useBandwidthEstimation: () => useSelector(store, (snapshot) => snapshot.bandwidthEstimation),
+    updateTrackMetadata: useSelector(store, (snapshot) => snapshot.connectivity.client?.updateTrackMetadata) ?? null,
+    setTrackBandwidth: useSelector(store, (snapshot) => snapshot.connectivity.client?.setTrackBandwidth) ?? null,
+    setTargetTrackEncoding:
+      useSelector(store, (snapshot) => snapshot.connectivity.client?.setTargetTrackEncoding) ?? null,
+    /* eslint-enable react-hooks/rules-of-hooks */
```

Comment:
`updateTrackMetadata`, `setTrackBandwidth`, `setTargetTrackEncoding` do not follow the 'useSth' naming convention. The names suggest that they are simple functions, but they depend on the use of a selector. If someone does not use them as hooks, they may not work. We should rename them.